server/Game.js

```diff
@@ -83,13 +87,16 @@ export default class {
     this.commands = [];
     const nextPlayer = this.nextPlayer();
     if (!nextPlayer) return;
+    if (this.history) this.history.close();
+    this.history = new History(this, options);
```

Comment:
this should maybe be set to null if the folder is null?

Reply:
Done